staging/src/k8s.io/legacy-cloud-providers/aws/aws_test.go

```diff
@@ -583,68 +583,43 @@ func testHasNodeAddress(t *testing.T, addrs []v1.NodeAddress, addressType v1.Nod
 	t.Errorf("Did not find expected address: %s:%s in %v", addressType, address, addrs)
 }
 
-func TestNodeAddresses(t *testing.T) {
-	// Note instance0 and instance1 have the same name
-	// (we test that this produces an error)
-	var instance0 ec2.Instance
-	var instance1 ec2.Instance
-	var instance2 ec2.Instance
-
-	// ClusterID needs to be set
+func makeInstance(num int, privateIP, publicIP, privateDNSName, publicDNSName string) ec2.Instance {
```

Comment:
nit: might be useful to send this refactoring as a separate PR (it's non-controversial and would make this commit pretty simple)

Reply:
Done, see [#91082](https://github.com/kubernetes/kubernetes/pull/91082)